frontend/src/views/edit-project/EditProjectNavbar/EditProjectNavbar.tsx

```diff
@@ -115,32 +105,20 @@ export const EditProjectNavbar = ({ subscribers }) => {
 
   let modal = null;
   if (modalDisplayed === 'CreateDocument') {
-    modal = <NewDocumentModal projectId={id} onDocumentCreated={onCloseModal} onClose={onCloseModal} />;
+    modal = <NewDocumentModal projectId={projectId} onDocumentCreated={onCloseModal} onClose={onCloseModal} />;
   } else if (modalDisplayed === 'UploadDocument') {
-    modal = <UploadDocumentModal projectId={id} onDocumentUploaded={onCloseModal} onClose={onCloseModal} />;
+    modal = <UploadDocumentModal projectId={projectId} onDocumentUploaded={onCloseModal} onClose={onCloseModal} />;
   } else if (modalDisplayed === 'RenameProject') {
     modal = (
       <RenameProjectModal
-        projectId={id}
+        projectId={projectId}
         initialProjectName={name}
         onProjectRenamed={onCloseModal}
         onClose={onCloseModal}
       />
     );
   } else if (modalDisplayed === 'DeleteProject') {
-    modal = <DeleteProjectModal projectId={id} onProjectDeleted={onProjectDeleted} onClose={onCloseModal} />;
-  }
-  let accessDetails = undefined;
-  if (!canEdit) {
-    accessDetails = (
-      <Text
-        className={styles.accessDetails}
-        data-testid="project-access-details">{`View Only (owned by ${projectOwner})`}</Text>
-    );
-  } else if (projectOwner !== username) {
-    accessDetails = (
-      <Text className={styles.accessDetails} data-testid="project-access-details">{`(owned by ${projectOwner})`}</Text>
-    );
```

Comment:
Is this expected that we lose this?

Reply:
Yes for now, I need to improve most of our toolbars and since we are not relying on useProject here, I'll have to update things to restore that feature.